producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/CxfConfig.java

```diff
@@ -0,0 +1,111 @@
+package uk.gov.moj.sdt.producers.comx.config;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import javax.xml.ws.Endpoint;
+
+import org.apache.cxf.Bus;
+import org.apache.cxf.bus.spring.SpringBus;
+import org.apache.cxf.ext.logging.LoggingFeature;
+import org.apache.cxf.jaxws.EndpointImpl;
+import org.apache.cxf.transport.servlet.CXFServlet;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.web.servlet.ServletRegistrationBean;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.moj.sdt.interceptors.in.SdtUnmarshallInterceptor;
+import uk.gov.moj.sdt.interceptors.in.ServiceRequestInboundInterceptor;
+import uk.gov.moj.sdt.interceptors.in.XmlInboundInterceptor;
+import uk.gov.moj.sdt.interceptors.out.CacheEndOutboundInterceptor;
+import uk.gov.moj.sdt.interceptors.out.CacheSetupOutboundInterceptor;
+import uk.gov.moj.sdt.interceptors.out.FaultOutboundInterceptor;
+import uk.gov.moj.sdt.interceptors.out.ServiceRequestOutboundInterceptor;
+import uk.gov.moj.sdt.interceptors.out.XmlOutboundInterceptor;
+import uk.gov.moj.sdt.interceptors.service.RequestDaoService;
+import uk.gov.moj.sdt.producers.comx.sdtws.SdtEndpointPortType;
+import uk.gov.moj.sdt.utils.concurrent.api.IInFlightMessage;
+
+@Configuration
+@EnableAutoConfiguration
+@ComponentScan("uk.gov.moj.sdt")
+@EnableCaching(proxyTargetClass = true)
+public class CxfConfig {
+
+    @Bean
+    public ServletRegistrationBean<CXFServlet> cxfServlet() {
+        return new ServletRegistrationBean<>(new CXFServlet(), "/producers-comx/service/*");
+    }
+
+    @Bean(name = Bus.DEFAULT_BUS_ID)
+    public SpringBus springBus(LoggingFeature loggingFeature) {
+        SpringBus cxfBus = new SpringBus();
+        cxfBus.getFeatures().add(loggingFeature);
+        return cxfBus;
+    }
```

Comment:
Do we need this?  I might be wrong, but I don't think sdt-commissioning uses a queue.

Reply:
Yes, we need this, this is not for service bus config